src/Umbraco.Core/IO/FileSystemProviderManager.cs

```diff
@@ -143,6 +147,7 @@ private void CreateWellKnownFileSystems()
         public IFileSystem2 XsltFileSystem { get { return _xsltFileSystem; } }
         public IFileSystem2 MasterPagesFileSystem { get { return _mvcViewsFileSystem; } }
         public IFileSystem2 MvcViewsFileSystem { get { return _mvcViewsFileSystem; } }
+        public IFileSystem2 LibFileSystem { get { return _libFileSystem; } }

```

Comment:
I think we should make this `internal` and renamed to `JavaScriptLibraryFileSystem` so devs know what it is, `LibFileSystem` doesn't really mean anything without some context.